Client/Frontend/TabContentsScripts/CreditCardHelper.swift

```diff
@@ -103,11 +103,15 @@ class CreditCardHelper: TabContentScript {
 
     func getFieldTypeValues(payload: CreditCardPayload) -> UnencryptedCreditCardFields {
         var ccPlainText = UnencryptedCreditCardFields()
+        let creditCardValidator = CreditCardValidator()
 
         ccPlainText.ccName = payload.ccName
         ccPlainText.ccExpMonth = Int64(payload.ccExpMonth.filter { $0.isNumber }) ?? 0
-        ccPlainText.ccExpYear = Int64(payload.ccExpYear.filter { $0.isNumber }) ?? 0
+        // Note: we only need the last 2 digits of the ccExpYear
+        ccPlainText.ccExpYear = (Int64(payload.ccExpYear.filter { $0.isNumber }) ?? 0) % 100
```

Comment:
Add Unit test for this function please

Reply:
Good call, let me add a test here for the sample card we have

Reply:
https://github.com/mozilla-mobile/firefox-ios/pull/14782/commits/8c8b28ec70fdbecd1f95654081935f76d939ede3